docs/server-side/v2/api-reference/external-streams/Get-details-for-a-job-id.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Get details for a given RTMP Job ID
```

Comment:
Is there a GET on /external-streams/ that returns a list of stream objects with filters for room id, session id and status?

Reply:
yes will be added in next release